docs/src/tutorials/standalone/Canopy/simple_canopy.jl

```diff
@@ -0,0 +1,115 @@
+# let's try doing the same with the canopy model.
+
+# This tutorial sets up our CanopyModel on a column domain.
+# The model prognoses canopy temperature and liquid water content,
+# and diagnoses many more variables.
+# Here you'll see that the setup for different models is very similar,
+# and we'll highlight the changes between them.
+
+# First import the Julia packages we'll need.
+import ClimaParams as CP
+using ClimaUtilities: TimeVaryingInput
+using ClimaLand
+using ClimaLand.Domains
+using ClimaLand.Canopy
+import ClimaLand.Simulations: LandSimulation, solve!
+import ClimaLand.Parameters as LP
+using Dates
+import ClimaDiagnostics
+using CairoMakie, ClimaAnalysis, GeoMakie, Poppler_jll, Printf, StatsBase
+import ClimaLand.LandSimVis as LandSimVis
+
+# Choose a floating point precision, and get the parameter set,
+# which holds constants used across CliMA models.
+FT = Float32
+earth_param_set = LP.LandParameters(FT);
+
+# We will run this simulation on a point domain at a random lat/lon location.
+# This is different from the soil example, which ran on a column, because the
+# soil model has depth and the canopy model does not.
+longlat = FT.((34.1, -118.1))
+domain = Domains.Point(; z_sfc = FT(0.0), longlat);
+surface_space = domain.space.surface;
+
+# We choose the initial and final simulation times as DatesTimes, and a timestep in seconds.
+start_date = DateTime(2008);
+end_date = start_date + Second(60 * 60 * 72);
+dt = 1000.0;
+
+# Whereas the soil model takes in 2 forcing objects (atmosphere and radiation),
+# the canopy takes in 3 (atmosphere, radiation, and ground). Here we read in the
+# first two from ERA5 data, and specify prescribed conditions for the ground.
+# We also set up a constant leaf area index (LAI).
+# This differs from the soil example because we have the extra inputs of the ground conditions and LAI.
+era5_ncdata_path =
+    ClimaLand.Artifacts.era5_land_forcing_data2008_path(; lowres = true);
+atmos, radiation = ClimaLand.prescribed_forcing_era5(
+    era5_ncdata_path,
+    surface_space,
+    start_date,
+    earth_param_set,
+    FT,
+);
+ground = PrescribedGroundConditions{FT}();
+LAI = TimeVaryingInput((t) -> FT(1.0));
+
+# Now, we can create the canopy model.
+# This constructor uses default parameters and parameterizations,
+# but these can also be overwritten, which we'll demonstrate in later tutorials.
+# Of course, this model construction differs from the soil example because we're
+# using a different model type, but the approach remains the same.
+model = Canopy.CanopyModel{FT}(
+    domain,
+    (; atmos, radiation, ground),
+    LAI,
+    earth_param_set,
+);
+
+# Define a function to set initial conditions for the prognostic variables.
+# Since these are specific to the model physics, the contents here differ from
+# the soil example, but the function structure remains the same.
+function set_ic!(Y, p, t0, model)
```

Comment:
We might mention that these variables and their meanings are described in the science tutorials?

Reply:
Good idea. Unfortunately there's no documentation for canopy energy so no description of T I believe.

Also, I know we just restructured the docs, but I think it would be helpful if the science documentation was accessible at the top-level - right now it seems kind of hidden away to me.